### PR TITLE
Fuck circuits

### PR DESCRIPTION
**Pull request by TDcoolguy300**
:cl: TDcoolguy300 AKA Gregg
del: Disabled Integrated Circuits codewise, if this was merged then nobody was on to fix the fucking config.
/:cl:

### CODE REVIEW

**Comment by Sarcalogo** [comment]
Why would you remove circuits?

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
merging due to the high CPU load it puts the server under

we can reevaluate adding them once the problem is identified

**Comment by GuyonBroadway** [comment]
this is also something to send upstream.

File: code/modules/integrated_electronics/core/printer.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	var/can_clone = FALSE		// Same for above, but will allow the printer to duplicate a specific assembly.
 	var/current_category = null
 	var/list/program			// Currently loaded save, in form of list
+	var/enabled = 0
```
Comment (ghost) [inline code/modules/integrated_electronics/core/printer.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

File: code/modules/integrated_electronics/core/printer.dm [hunk 2]
```diff
@@ -130,7 +131,7 @@
 		new build_type(get_turf(loc))
 
 	if(href_list["print"])
-		if(!CONFIG_GET(flag/ic_printing))
+		if(!CONFIG_GET(flag/ic_printing) || !enabled)
```
Comment (ghost) [inline code/modules/integrated_electronics/core/printer.dm]: You've made changes to /tg/ files, please mark these changes.

**All modifications to non-hippie files should be marked.**

- Multi line changes start with `// hippie start` and end with `// hippie end`
- You can put a messages with a change if it isn't obvious, like this: `// hippie start - reason`
  - Should generally be about the reason the change was made, what it was before, or what the change is
  - Multi-line messages should start with `// hippie start` and use `/* Multi line message here */` for the message itself
- Single line changes should have `// hippie` or `// hippie - reason`

If you need to mirror a file, or function into a hippie-specific file, please leave behind a comment stating where it went.

```
// hippie start - Mirrored this function in <file> for <reason>
bunch of shitcode here
// hippie stop
```

Once you mirror a file, please follow the above for marking your changes, this way we know what needs to be updated when a file has been mirrored.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Perhaps someone making shittons of circuits is the problem, and said people need to be warmed?

**Comment by Iamgoofball** [comment]
you're a retard if you think this prevents circuits lmao

**Comment by ghost** [comment]
Do it.  And switch host box to EU faggots

**Comment by ghost** [comment]
MERGE

**Comment by ghost** [comment]
Closing this PR as it literally does nothing

**Pull request closed without merging**